jetstream/experimenter.py

```diff
@@ -158,20 +161,44 @@ class ExperimentCollection:
     # for nimbus experiments
     EXPERIMENTER_API_URL_V4 = "https://experimenter.services.mozilla.com/api/v4/experiments/"
 
+    @classmethod
+    def _handle_errors(cls, func: Callable[[], Any], slug: str) -> Optional[Experiment]:
+        """Handles exceptions when pulling in and working with experiments from Experimenter."""
+        try:
+            return func()
+        except Exception as e:
+            logger.exception(str(e), exc_info=e, extra={"experiment": slug})
+            return None
+
     @classmethod
     def from_experimenter(cls, session: requests.Session = None) -> "ExperimentCollection":
         session = session or requests.Session()
         legacy_experiments_json = session.get(cls.EXPERIMENTER_API_URL_V1).json()
+
         legacy_experiments = [
-            ExperimentV1.from_dict(experiment).to_experiment()
+            e
             for experiment in legacy_experiments_json
             if experiment["type"] != "rapid"
+            and (
+                e := cls._handle_errors(
```

Comment:
a walrus in a list comprehension might be a bridge too far :) for clarity's sake can we make these explicit loops with a try/except? i don't think we have to worry too much about DRY for n=2 since we're not handling them exactly the same anyway.